### DISCUSSION TITLE
Disable dependabot via YAML

### DISCUSSION THREAD

**Discussion by sandstrom** [dependabot/dependabot-core#8960] [Ideas]
We want to configure dependabot labels (via `.github/dependabot.yml`), so that when we manually trigger an update via the security tab, the PR will use the correct labels.

However, we don't want to run dependabot on a schedule.

For us, it would be great if we could disable it using one of the following:

- `schedule.interval` was optional, and wouldn't run if it wasn't set or
- `schedule.interval` could take the value `'never'`.

Related issue: https://github.com/dependabot/dependabot-core/issues/8116

**Comment by yeikel** [comment]
I think that if we have an issue, we should continue the conversation there as having the same conversation in two places doesn't help
